README.md

```diff
@@ -4,14 +4,20 @@
 
 Webhooks allow you to build or set up integrations which subscribe to certain events on ActiGraph's CentrePoint ecosystem. When one of those events is triggered, we'll send a HTTP POST payload to the webhook's configured URL. Webhooks can be used to update an external system or just for simple notifications.
```

Comment:
Let's change the language here to remove "we'll" - "we'll send a HTTP POST payload to the webhook's configured URL"